desc/objectives/objective_funs.py

```diff
@@ -558,7 +544,7 @@ def compute_scalar(self, *args, **kwargs):
     def print_value(self, *args, **kwargs):
         """Print the value of the objective."""
         x = self._unshift_unscale(self.compute(*args, **kwargs))
-        print(self._print_value_fmt.format(jnp.linalg.norm(x)))
+        logging.warning(self._print_value_fmt.format(jnp.linalg.norm(x)))
```

Comment:
Gotcha.  I've altered several of the print-formating function in the utils modules like pretty_print and the timer printing functions, because they seem like things desc primarily calls, rather than the user.  Are there other functions like this meant specifically for the user to output to stdout as well that I should revert?